src/jquery-sticky-header-footer.js

```diff
@@ -60,8 +61,17 @@
                 /**
                     Remove added DOM elements and plugin data
                  */
-                $('.' + classNames.outerWrapper).before(element).remove();
-                element.removeData('plugin_' + pluginName);
+                $.each(['footerElement', 'headerElement'], function(idx, val) {
```

Comment:
I am not following this. Seems like it should be:

    $.each([this.footerElement, this.headerElement], function(idx, val) {